tests/test-plugin.h

```diff
@@ -41,50 +41,66 @@ struct plugin_call_count
          */
         //@{
         int new_connection;
+        int connection_established;
+        int connection_closed;
         int new_address;
+        int address_removed;
         int new_subflow;
         int subflow_closed;
-        int connection_closed;
+        int subflow_priority;
         //@}
 };
 
 inline void plugin_call_count_reset(struct plugin_call_count *p)
 {
-        p->new_connection    = 0;
-        p->new_address       = 0;
-        p->new_subflow       = 0;
-        p->subflow_closed    = 0;
-        p->connection_closed = 0;
+        p->new_connection         = 0;
+        p->connection_established = 0;
+        p->connection_closed      = 0;
+        p->new_address            = 0;
+        p->address_removed        = 0;
+        p->new_subflow            = 0;
+        p->subflow_closed         = 0;
+        p->subflow_priority       = 0;
 }
 
-inline bool plugin_call_count_is_sane(struct plugin_call_count const *lhs,
-                                      struct plugin_call_count const *rhs)
+inline bool plugin_call_counts_are_pos(struct plugin_call_count const *p)
 {
-        return     lhs->new_connection >= 0
-                && lhs->new_address >= 0
-                && lhs->new_subflow >= 0
-                && lhs->subflow_closed >= 0
-                && lhs->connection_closed >= 0
-
-                && rhs->new_connection >= 0
-                && rhs->new_address >= 0
-                && rhs->new_subflow >= 0
-                && rhs->subflow_closed >= 0
-                && rhs->connection_closed >= 0
-
-                && lhs->connection_closed <= rhs->new_connection
-                && lhs->subflow_closed <= rhs->new_subflow;
+        return     p->new_connection >= 0
+                && p->connection_established >= 0
+                && p->connection_closed >= 0
+                && p->new_address >= 0
```

Comment:
I just noticed a missing `&& p->address_removed >= 0` condition here.  I'll add it.